SabreTools.Serialization/Readers/ZSTD.cs

```diff
@@ -0,0 +1,64 @@
+using System.IO;
+using SabreTools.Data.Models.ZSTD;
+using SabreTools.IO.Extensions;
+using static SabreTools.Data.Models.ZSTD.Constants;
+
+namespace SabreTools.Serialization.Readers
+{
+    public class ZSTD : BaseBinaryReader<Header>
+    {
+        /// <inheritdoc/>
+        public override Header? Deserialize(Stream? data)
+        {
+            // If the data is invalid
+            if (data == null || !data.CanRead)
+                return null;
+
+            try
+            {
+                // Cache the current offset
+                long initialOffset = data.Position;
+
+                #region Header
+                
+                var header = ParseHeader(data);
+
+                if (header == null)
```

Comment:
Minor nit: Just remove the extra newline here

Reply:
Fixed